src/fetch/payouts.py

```diff
@@ -270,7 +273,13 @@ def extend_payment_df(pdf: DataFrame, converter: TokenConversion) -> DataFrame:
     pdf["reward_eth"] = pdf["payment_eth"] - pdf["execution_cost_eth"]
     pdf["reward_cow"] = pdf["reward_eth"].apply(converter.eth_to_token)
 
-    secondary_allocation = max(PERIOD_BUDGET_COW - pdf["reward_cow"].sum(), 0)
+    secondary_allocation = max(
+        min(
+            PERIOD_BUDGET_COW - pdf["reward_cow"].sum(),
+            converter.eth_to_token(CONSISTENCY_REWARD_CAP),
+        ),
+        0,
```

Comment:
```suggestion
        0
```

Reply:
This is due to black (where multiline tuples always have a trailing comma).